.github/workflows/release.yml

```diff
@@ -32,36 +24,26 @@ jobs:
           force: ${{ github.event.inputs.force }}
       - uses: actions/checkout@v2
         with:
-          token: ${{ secrets.GH_SENTRY_BOT_PAT }}
-          fetch-depth: 20
+          token: ${{ secrets.GH_RELEASE_PAT }}
       - uses: getsentry/craft@master
         name: Craft Prepare
-        if: ${{ !github.event.inputs.skip_prepare }}
         with:
           action: prepare
           version: ${{ env.RELEASE_VERSION }}
         env:
           ZEUS_API_TOKEN: ${{ secrets.ZEUS_API_TOKEN }}
```

Comment:
Why do we still need the Zeus token here? Isn't Zeus used for publishing, and we're no longer publishing from here?

Reply:
~~Zeus is also to upload artifacts but let me check.~~

Seems like you are right, I'll axe it too.